src/platform/packages/shared/kbn-tracing/src/otlp_span_processor.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import type { OTLPExportConfig } from '@kbn/tracing-config';
+import { OTLPTraceExporter as OTLPTraceExporterHTTP } from '@opentelemetry/exporter-trace-otlp-http';
+import { OTLPTraceExporter as OTLPTraceExporterProto } from '@opentelemetry/exporter-trace-otlp-proto';
```

Comment:
Q: Should we use `@opentelemetry/exporter-trace-otlp-grpc` instead?

https://www.npmjs.com/package/@opentelemetry/exporter-trace-otlp-grpc?activeTab=dependencies

Maybe @trentm @david-luna can weigh in?